src/Action/BaseCreateAction.php

```diff
@@ -0,0 +1,131 @@
+<?php
+namespace Crud\Action;
+
+use Crud\Event\Subject;
+use Crud\Traits\FindMethodTrait;
+use Crud\Traits\RedirectTrait;
+use Crud\Traits\SaveMethodTrait;
+use Crud\Traits\SerializeTrait;
+use Crud\Traits\ViewTrait;
+use Crud\Traits\ViewVarTrait;
+
+/**
+ * Base Create/Update Crud class
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ */
+abstract class BaseCreateAction extends BaseAction
```

Comment:
i dislike the `BaseCreateAction` name, especially when an edit class suddenly extends a Create class - doesn't make sense to me - currently I'm loss for a better name, but BaseCreateAction wont fly imo